spec/order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "order"
+
+describe Order do
+  subject(:order) { described_class.new }
+  let(:menu) { double(:menu) }
+
+  before do
+    order.select_items("rice", 2)
+  end
+
+  it "should allow a customer to order a number of items" do
+    expect(order.selection).to eq({ :rice => 2 })
+  end
+
+  it "should add the selection to the order" do
+    expect(order.shopping_cart).to eq([{ :rice => 2 }])
+  end
+
+  it "should get the price of each item in the order" do
+    expect(order.work_out_price).to eq([{ :rice => 2 }])
+  end
+
+  context "#price_calculation" do
```

Comment:
`context` is used to describe the conditions that you're testing under. E.g. if you want to test whether your food-pipe still works when you're upside-down, you might have the `context "when I'm upside-down"`.
`describe` means 'I'm about to describe the behaviour of the following method or a class'.
But they both actually _do_ the same thing.